README.md

```diff
@@ -4,7 +4,69 @@ An API for friendly boolean parsing and handling
 
 ## Usage
 
-TODO: Write code samples and usage instructions here
+```ruby
+require 'much-boolean'
+MuchBoolean::FALSE_VALUES # => [nil, 0, "0",
+                                false, "false", "False", "FALSE", "f", "F",
+                                "no", "No", "NO", "n", "N"]
+
+# nils, ones/zeros type values
+MuchBoolean.parse(nil) # => false
+MuchBoolean.parse(0)   # => false
+MuchBoolean.parse('0') # => false
+MuchBoolean.parse(1)   # => false
+MuchBoolean.parse('1') # => false
```

Comment:
@kellyredding - I think you want these last 2 (the 1's) to be true instead of false.